src/archivematicaCommon/lib/databaseFunctions.py

```diff
@@ -106,53 +203,72 @@ def logTaskCreatedSQL(taskManager, commandReplacementDic, taskUUID, arguments):
-         taskexec, arguments, databaseInterface.getUTCDate())
-    )
-
-def logTaskAssignedSQL(taskUUID, client, date):
-    databaseInterface.runSQL(
-        """UPDATE Tasks SET startTime=%s, client=%s WHERE taskUUID=%s;""",
-        (date, client, taskUUID)
-    )
+    Task.objects.create(taskuuid=taskUUID,
+                        job_id=jobUUID,
+                        fileuuid=fileUUID,
+                        filename=fileName,
+                        execution=taskexec,
+                        arguments=arguments,
+                        createdtime=getUTCDate())
 
 def logTaskCompletedSQL(task):
+    """
+    Fetches execution data from the completed task and logs it to the database.
+    Updates the entry in the Tasks table with data in the provided task.
+    Saves the following fields: exitCode, stdOut, stdError
+
+    :param task:
+    """
     print "Logging task output to db", task.UUID
     taskUUID = task.UUID.__str__()
     exitCode = task.results["exitCode"].__str__()
     stdOut = task.results["stdOut"]
     stdError = task.results["stdError"]
 
-    databaseInterface.runSQL(
-        """UPDATE Tasks SET endTime=%s, exitCode=%s, stdOut=%s, stdError=%s
-           WHERE taskUUID=%s""",
-        (databaseInterface.getUTCDate(), exitCode, stdOut, stdError, taskUUID)
-    )
+    task = Task.objects.get(taskuuid=taskUUID)
+    task.endtime = getUTCDate()
+    task.exitcode = exitCode
+    task.stdout = stdOut
+    task.stderror = stdError
+    task.save()
 
 def logJobCreatedSQL(job):
-    separator = databaseInterface.getSeparator()
+    """
+    Logs a job's properties into the Jobs table in the database.
+
+    :param jobChainLink job: A jobChainLink instance.
+    :returns None:    
+    """
     unitUUID =  job.unit.UUID
-    decDate = databaseInterface.getDeciDate("." + job.createdDate.split(".")[-1])
+    decDate = getDeciDate("." + job.createdDate.split(".")[-1])
     if job.unit.owningUnit != None:
         unitUUID = job.unit.owningUnit.UUID 
-    databaseInterface.runSQL("""INSERT INTO Jobs (jobUUID, jobType, directory, SIPUUID, currentStep, unitType, microserviceGroup, createdTime, createdTimeDec, MicroServiceChainLinksPK, subJobOf)
-        VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)""",
-        (job.UUID, job.description, job.unit.currentPath, unitUUID,
-         "Executing command(s)", job.unit.__class__.__name__,
-         str(job.microserviceGroup), job.createdDate, decDate, str(job.pk),
-         str(job.subJobOf))
-    )
-    #TODO -un hardcode executing exeCommand
-
-def logJobStepCompletedSQL(job):
-    databaseInterface.runSQL("""INSERT INTO jobStepCompleted (jobUUID, step, completedTime)
-        VALUES (%s, %s, %s)""",
-        (job.UUID, job.step, databaseInterface.getUTCDate())
-    )
-
-def fileWasRemoved(fileUUID, utcDate=databaseInterface.getUTCDate(), eventDetail = "", eventOutcomeDetailNote = "", eventOutcome=""):
+    job = Job(jobuuid=job.UUID,
```

Comment:
Ouch, good catch. I'll just not assign the new job to a variable since it's never used/returned.